src/generators/android/templates/abstractSerializableClass.java.hbs

```diff
@@ -0,0 +1,8 @@
+package com.segment.generated;
+
+import java.util.*;
+import com.segment.analytics.Properties;
+
+abstract class SEGSerializable {
```

Comment:
Let's drop the `SEG` prefix here. We only need these namespaces in Objective-C land :)

Reply:
Gotcha - I added the `SEG` prefix because Java has a public interface `Serializable`. So as to avoid confusion, I'll change this to `SerializableProperties`.